undeclared ZEND_JMPZNZ

I tried to compile parallel on my old raspberry 2 model B using the docker image php:8.3-zts and got the following error:

548.1 /bin/bash /tmp/pear/temp/pear-build-defaultuserRQMTMs/parallel-1.2.1/libtool --mode=compile cc -I. -I/tmp/pear/temp/parallel -I/tmp/pear/temp/pear-build-defaultuserRQMTMs/parallel-1.2.1/include -I/tmp/pear/temp/pear-build-defaultuserRQMTMs/parallel-1.2.1/main -I/tmp/pear/temp/parallel -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/tmp/pear/temp/parallel  -DHAVE_CONFIG_H  -g -O2 -D_GNU_SOURCE   -Wall -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1  -DZEND_COMPILE_DL_EXT=1 -c /tmp/pear/temp/parallel/src/cache.c -o src/cache.lo  -MMD -MF src/cache.dep -MT src/cache.lo
549.0  cc -I. -I/tmp/pear/temp/parallel -I/tmp/pear/temp/pear-build-defaultuserRQMTMs/parallel-1.2.1/include -I/tmp/pear/temp/pear-build-defaultuserRQMTMs/parallel-1.2.1/main -I/tmp/pear/temp/parallel -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/tmp/pear/temp/parallel -DHAVE_CONFIG_H -g -O2 -D_GNU_SOURCE -Wall -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -DZEND_COMPILE_DL_EXT=1 -c /tmp/pear/temp/parallel/src/cache.c -MMD -MF src/cache.dep -MT src/cache.lo  -fPIC -DPIC -o src/.libs/cache.o
551.4 /tmp/pear/temp/parallel/src/cache.c: In function 'php_parallel_cache_create':
551.4 /tmp/pear/temp/parallel/src/cache.c:247:22: error: 'ZEND_JMPZNZ' undeclared (first use in this function); did you mean 'ZEND_JMPNZ'?
551.4   247 |                 case ZEND_JMPZNZ:
551.4       |                      ^~~~~~~~~~~
551.4       |                      ZEND_JMPNZ
551.4 /tmp/pear/temp/parallel/src/cache.c:247:22: note: each undeclared identifier is reported only once for each function it appears in
552.2 make: *** [Makefile:239: src/cache.lo] Error 1
552.2 ERROR: `make' failed

I think ZEND_JMPZNZ should be removed as of php version 8.3 since i could not find any reference in the php master branch on github. for me the build worked after removing it.

--- a/src/cache.c
+++ b/src/cache.c
@@ -244,7 +244,9 @@ static zend_op_array* php_parallel_cache_create(const zend_function *source, zen
                 case ZEND_FAST_CALL:
                     opline->op1.jmp_addr = &opcodes[opline->op1.jmp_addr - source->op_array.opcodes];
                 break;
+#if PHP_VERSION_ID < 80200
                 case ZEND_JMPZNZ:
+#endif
                 case ZEND_JMPZ:
                 case ZEND_JMPNZ:
                 case ZEND_JMPZ_EX:

